src/lib/components/forms/form/form-components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input } from '@angular/core';
+import { TerraPlacementEnum } from '../../../../../helpers';
+import { noop } from 'rxjs';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { MultiSelectInterface } from './multi-select.interface';
+import { TerraMultiCheckBoxValueInterface } from '../../../multi-check-box/data/terra-multi-check-box-value.interface';
+
+@Component({
+    selector: 'tc-multi-select',
+    templateUrl: './multi-select.component.html',
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: MultiSelectComponent,
+            multi: true
+        }
+    ]
+})
+export class MultiSelectComponent implements ControlValueAccessor, MultiSelectInterface {
+    /** The name of the control which will be used as label. */
+    @Input()
+    public name: string = '';
+
+    /** Disables the select when set to true. Default false. */
+    @Input()
+    public isDisabled: boolean = false;
+
+    /** Requires the select to be filled when set to true. Default false. */
+    @Input()
+    public isRequired: boolean = false;
+
+    /** Text that should be shown in a tooltip on the control. */
+    @Input()
+    public tooltipText: string = '';
+
+    /** Set the tooltip placement (bottom, top, left, right). Default top. */
+    @Input()
+    public tooltipPlacement: TerraPlacementEnum = TerraPlacementEnum.TOP;
+
+    /** List of options that the user can select from. */
+    @Input()
+    public listBoxValues: Array<TerraMultiCheckBoxValueInterface> = [];
+
+    /** Internal model. Stores the value of the selected option. */
+    public value: Array<TerraMultiCheckBoxValueInterface>;
```

Comment:
Das ist die Liste der ausgewählten Werte, daher
```suggestion
    public value: Array<any>;
```